tensorflow/python/ops/image_ops_impl.py

```diff
@@ -5581,6 +5581,11 @@ def _sort_scores_and_boxes(scores, boxes):
       x_min, x_max = control_flow_ops.cond(
           x_1_is_min, lambda: (x_1, x_2), lambda: (x_2, x_1))
       boxes = array_ops.concat([y_min, x_min, y_max, x_max], axis=2)
+  else:
+    #TODO(mdanatg/bhack) Without initzializing control_flow_ops.cond variable
+    # in the else branch it is failing autograph control_flow operator
+    # in aug_orelse() verify_tf_cond_branch_ check
+    y_1, x_1, y_2, x_2 = None, None, None, None
```

Comment:
@mdanatg This is just quick hack to pass the test. 
I suppose this emerged as now variables captured in lamba are `live_out` so `_verify_tf_cond_branch_ check` in the autograph operator has a constrain check to have these in the else branch.